refactor(ivy): remove load instruction

crisbeto opened the pull request:
These changes remove the `ɵɵload` instruction which isn't being generated anymore.

alfaproject commented on packages/core/test/render3/query_spec.ts:
```diff
@@ -1905,8 +1908,9 @@ describe('query', () => {
-                 deepInstance = ɵɵload<DeepQueryDirective>(2);
+                 const lView = getLView();
+                 shallowInstance = loadInternal<ShallowQueryDirective>(lView, 1);
+                 deepInstance = loadInternal<DeepQueryDirective>(lView, 2);
```
Isn't this an external surface?

pkozlowski-opensource replied:
Nope, it is just used here since it was the easiest way to grab an instance of a directive in the hand-written "generate" code. We are slowly phasing out those hand-written tests so this particular usage will go away soon(ish).

pkozlowski-opensource reviewed (approved):
LGTM, thnx @crisbeto - it was on my TODO list! 

If you into those type of changes we could also replace `ɵɵloadViewQuery` and ` ɵɵloadContentQuery` with just one instruction (` ɵɵloadQuery` ?) as the 2 mentioned functions are the same now.

crisbeto commented:
Cool! Will do next week.

AndrewKushnir commented:
- [VE Presubmit](http://test/OCL:262581434:BASE:262581463:1565370533710:5f748e01)
- [Ivy Preview](http://test/OCL:262581434:BASE:262581694:1565370600938:3c8d5756)

kara commented on packages/core/src/render3/i18n.ts:
```diff
@@ -912,7 +912,7 @@ function removeNode(index: number, viewData: LView) {
   }
 
-  const slotValue = ɵɵload(index) as RElement | RComment | LContainer;
+  const slotValue = loadInternal(viewData, index) as RElement | RComment | LContainer;
```
Since there's no longer a `loadExternal`, can we rename `loadInternal` to `load`?

kara reviewed (approved):
LGTM, one minor thing

crisbeto committed:
refactor(ivy): remove load instruction

These changes remove the `ɵɵload` instruction which isn't being generated anymore.

crisbeto commented:
I've reworked in based on the feedback @kara.

kara reviewed (approved):
LGTM

kara commented:
[VE presubmit](https://test.corp.google.com/ui#id=OCL:262973866:BASE:262973905:1565636017087:1464b061)

[Ivy presubmit](https://test.corp.google.com/ui#id=OCL:262973866:BASE:262974099:1565636070172:ca453449)

kara commented:
merge-assistance: global approval
